applications/asset_tracker_v2/doc/app_module.rst

```diff
@@ -35,7 +35,7 @@ Application start
 =================
 
 The application module is the system's initial point of entry and the source file of the module is :file:`asset_tracker_v2/src/main.c`.
-When the application boots, the module initializes the :ref:`event_manager` and sends out the initial event :c:enum:`APP_EVT_START` that starts the rest of the modules in the application.
+When the application boots, the module initializes the :ref:`app_evt_mgr` and sends out the initial event :c:enum:`APP_EVT_START` that starts the rest of the modules in the application.
```

Comment:
Should we also update refs in the old release notes? Otherwise they would not work.